index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
+        <a href=""><img src="images/Union.png" alt="" /></a>
+      </nav>
+    </header>
+    <div class="heading-section">
```

Comment:
- [ ]   I would suggest you kindly use a `<section>` tag to start a new section of your project which will have all other `<div>` wrapped inside it. As this represents a generic standalone section of the document. Kindly check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.